lib/Makefile.msvc

```diff
@@ -12,16 +12,16 @@
 #
 THIS_MAKEFILE := $(lastword $(MAKEFILE_LIST))
 
-USE_CYTHON := 1
-#USE_CYTHON := 0
+USE_CYTHON := 0
+#USE_CYTHON := 1
 
-_VERSION    := $(shell grep AC_INIT ../configure.ac | cut -d'[' -f3 | sed -r -e 's/(-DEV)?], //g')
+_VERSION	:= $(shell grep AC_INIT ../configure.ac | cut -d'[' -f3 | sed -e 's/-DEV//g' -e 's/], //g')
 _VERSION   	:= $(subst ., ,$(_VERSION))
 VER_MAJOR  	:= $(word 1,$(_VERSION))
 VER_MINOR  	:= $(word 2,$(_VERSION))
 VER_MICRO   := $(word 3,$(_VERSION))
 VERSION     := $(VER_MAJOR).$(VER_MINOR).$(VER_MICRO)
-VERSION_NUM := ($(VER_MAJOR) << 16) + ($(VER_MINOR) << 8) + $(VER_MICRO)
+VERSION_NUM	:= 0x$(shell if [ ${VER_MAJOR} -lt 10 ] ;then echo 0;fi)$(VER_MAJOR)$(shell if [ ${VER_MINOR} -lt 10 ] ;then echo 0;fi)$(VER_MINOR)$(shell if [ ${VER_MICRO} -lt 10 ] ; then echo 0;fi)$(VER_MICRO)

```

Comment:
Since VER_MAJOR is decimal, when it reaches 10, this becomes 0x10, which is 16 in decimal.   So if you'd like to use 0x, you need to convert VER_MAJOR to hex string.
Of course, VER_MAJOR is not going to get bigger, but VER_MINOR could be.